app/components/forms/user-profile-form.js

```diff
@@ -41,20 +40,6 @@ export default Component.extend(FormMixin, {
               prompt : this.l10n.t('Please enter a valid email ID')
             }
           ]
-        },
-        phone: {
-          identifier : 'phone',
-          rules      : [
-            {
-              type   : 'empty',
-              prompt : this.l10n.t('Please enter a phone number.')
-            },
-            {
-              type   : 'regExp',
-              value  : validPhoneNumber,
```

Comment:
why remove this?

Reply:
I think that after removing an "empty" type, a validator can't allow empty input in the phone field. so I just remove it and add inputtype=number.